modules/api/command/CommandManager.cpp

```diff
@@ -14,8 +15,10 @@ void CommandManager::process(MessageEvent& event) {
 	std::getline(ss, command, ' ');
 
 	it = cspecs.find(command);
-	if (it == cspecs.end())
-		throw CommandNotFoundException();
```

Comment:
`CommandNotFoundException` now unused, file can be deleted (check for ohter exceptions if if they are used).